doc/api/n-api.md

```diff
@@ -6267,6 +6269,95 @@ idempotent.
 
 This API may only be called from the main thread.
 
+## Using Node.js as a shared library (`libnode`)
```

Comment:
This has nothing to do with shared library mode.

Reply:
This API allows using `libnode` through Node-API and it has no other uses.

Reply:
That’s … wrong? Embedding and usage as a shared library are conceptually related but technically orthogonal.

Reply:
Ok, let's call it `Using an embedded version of Node.js`